.travis.yml

```diff
@@ -39,9 +39,9 @@ before_script:
   - git config --global user.name "Travis CI"
 
 script:
-  - xdebug-enable
-  - vendor/bin/phpunit --coverage-clover=build/logs/clover.xml
-  - xdebug-disable
+  - if [[ "$WITH_COVERAGE" == "true" ]]; then xdebug-enable; fi
```

Comment:
It doesn't, but I extracted a function, see [`.travis/travis.sh`](https://github.com/codeclimate/php-test-reporter/pull/95/files#diff-02b12a62ed710ace978372c0e5250b01)!